### DISCUSSION TITLE
%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%983/

### DISCUSSION THREAD

**Discussion by giscus** [SungbinYang/SungbinYang.github.io#7] [Comments]
# %EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%983/

양성빈의 블로그

https://sungbin.kr/%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%983/

**Comment by ljh468** [comment]
문제풀이 잘 봤습니다.!!
고급 3번 정답 함수는 굉장히 이븐하게 익은 함수네요 :)

**Reply by SungbinYang** [reply]
ㅋㅋㅋㅋㅋㅋ 감사합니다! 굉장히 이븐하죠? ㅋㅋㅋ

**Reply by lazyshu** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋ 야미
